### PR TITLE
Implemented unit test for ProfileContainerMobile component

### PR DESCRIPTION
**Pull request by TetianaDeresh**
image

### CODE REVIEW

File: tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    defaultQuantity: 2,
+    doneItems: [
+      { title: 'Item 1', description: 'Description 1' },
+      { title: 'Item 2', description: 'Description 2' }
+    ],
+    chipItems: ['Subject 1', 'Subject 2']
+  }
+
+  it('renders user information', () => {
+    const { getByText } = render(<ProfileContainerMobile {...mockProps} />)
```
Comment (mxrcury) [inline tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx:25]: you can add this render to `beforeEach` after mockProps and use `getByText` as method of the `screen` imported from `@testing-library/react`

File: tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    expect(getByText('John Doe')).toBeInTheDocument()
+    expect(getByText('Experienced tutor')).toBeInTheDocument()
+  })
+
+  it('renders action icon', () => {
+    const { getByText } = render(<ProfileContainerMobile {...mockProps} />)
```
Comment (mxrcury) [inline tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx:32]: separate "expects" by whitespace line and the same below

File: tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    actionIcon: <button>Edit</button>,
+    accInfo: <div>Account info</div>,
+    buttonGroup: <div>Button group</div>,
+    defaultQuantity: 2,
+    doneItems: [
+      { title: 'Item 1', description: 'Description 1' },
+      { title: 'Item 2', description: 'Description 2' }
+    ],
+    chipItems: ['Subject 1', 'Subject 2']
+  }
```
Comment (Yurenko) [inline tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx:22]: Move out mock data from describe on top level

Comment (TetianaDeresh) [reply tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx]: Fixed it

File: tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx (hunk 2 above)
Comment (TetianaDeresh) [reply tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx]: Fixed it

File: tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx (hunk 1 above)
Comment (TetianaDeresh) [reply tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx]: Fixed it

File: tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  })
+
+  it('renders done items', () => {
+    expect(screen.getByText('Item 1')).toBeInTheDocument()
+
+    expect(screen.getByText('Description 1')).toBeInTheDocument()
+
+    expect(screen.getByText('Item 2')).toBeInTheDocument()
+
+    expect(screen.getByText('Description 2')).toBeInTheDocument()
```
Comment (Tolik170) [inline tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx:51]: you don`t need to check for every item in this and another test cases, you can check only for item 1 and desription 1

File: tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  })
+
+  it('renders account info and button group', () => {
```
Comment (Tolik170) [inline tests/unit/containers/tutor-profile/profile-info/ProfileContainerMobile.spec.jsx:54]: Also, try to test click on show more button for done elements

**Review by maarekua — APPROVED** [review]

**Review by mxrcury — APPROVED** [review]

**Review by TSlashDreamy — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request closed without merging**